sdk/storage/Azure.Storage.Queues/tests/MessageIdClientTests.cs

```diff
@@ -86,7 +86,7 @@ public async Task UpdateAsync_Update()
                 var result = await messageId.UpdateAsync(message1, enqueuedMessage.PopReceipt, new TimeSpan(100));
 
                 // Assert
-                Assert.IsNotNull(result.Raw.Headers.RequestId);
+                Assert.IsNotNull(result.GetRawResponse().Headers.RequestId);
```

Comment:
Was this accidentally left in from the last change?

Reply:
yeah, undone.